src/ui/ResponsiveComponents/ResponsiveComponentsManager.ts

```diff
@@ -143,7 +143,7 @@ export class ResponsiveComponentsManager {
         }
       },
       ResponsiveComponentsManager.RESIZE_DEBOUNCE_DELAY,
-      true
+      false
```

Comment:
If the default is false can we just leave it out?

Reply:
Ok I will remove it, I wasn't sure if there was any value in explicitly setting it.